src/sentry/static/sentry/app/components/checkboxFancy.tsx

```diff
@@ -18,7 +25,7 @@ const getHoverStyles = p =>
     border: 2px solid ${p.checked ? p.theme.purple : p.theme.gray4};
   `;
 
-const CheckboxFancy = styled(({checked, disabled, ...props}) => (
+const CheckboxFancy = styled(({checked, disabled, ...props}: Props) => (
   <div role="checkbox" aria-disabled={disabled} aria-checked={checked} {...props}>
```

Comment:
Looks like we spread into a div, this should probably include `HtmlDivElement` props right?

Reply:
We never actually use the spread for anything productive. I'm going to remove the spread and pass the className prop down instead.

Reply:
Okay